features/admin.roles.v2/models/policies.ts

```diff
@@ -16,5 +16,16 @@
  * under the License.
  */
 
-export * from "./application-roles";
-export * from "./api-authorization/api-authorization";
+/**
+ * Policies enum.
+ */
+export enum Policy {
+    /**
+     * Role based policy.
+     */
+    ROLE = "RBAC",
+    /**
+     * No Policy.
+     */
+    NO = "NONE"
```

Comment:
WDYT?
```suggestion
    NONE = "NONE"
```

Reply:
This was an already existing model and seems like moving the file and renaming has messed up the Git history.